src/DynamoCoreWpf/Services/IconsService.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Resources;
+using System.Text;
+using System.Windows;
+using System.Windows.Media.Imaging;
+using Dynamo.DSEngine;
+
+namespace Dynamo.Wpf.Services
+{
+    public class IconsService
+    {
+        private static Dictionary<Assembly, IconsWarehouse> warehouses =
+            new Dictionary<Assembly, IconsWarehouse>();
```

Comment:
Hi @vmoyseenko, I'd prefer if we do not make this `static`, can we not place an instance of `IconsService` under `DynamoViewModel`?